### PR TITLE
[ES] Expansion Content

### PR DESCRIPTION
**Pull request by luischos**
expansion_content1
expansion_content2

### CODE REVIEW

File: translations/expansion_content.tex/es.po [hunk 1]
```diff
@@ -84,7 +84,8 @@ msgstr ""
 #. type: Plain text
 #: sections/expansion_content.tex:3
 msgid "\\addsection{Expansion Content}{\\skills/pathfinding.png}"
-msgstr ""
+msgstr "\\addsection{Ampliación de Contenido}{\\skills/pathfinding.png}\n"
```
Comment (qwrtln) [inline translations/expansion_content.tex/es.po:87]: Argh, this will be a pain to fix...

Comment (luischos) [reply translations/expansion_content.tex/es.po:87]: Yes, I saw it... Is possible do it in 2 rows?

Comment (luischos) [reply translations/expansion_content.tex/es.po]: Changed, better now @qwrtln

**Comment by luischos** [comment]
Can you approve it please? @qwrtln

**Review by qwrtln — APPROVED** [review]

**Pull request merged**